app/routes/approve-application-claim.js

```diff
@@ -11,13 +11,25 @@ module.exports = {
     pre: [{ method: preDoubleSubmitHandler }],
     validate: {
       payload: Joi.object({
-        approveClaim: Joi.string().valid('yes', 'no'),
+        confirm: Joi.array().items(Joi.string().valid('approveClaim', 'sentChecklist')).required(),
         reference: Joi.string().valid(),
         page: Joi.number().greater(0).default(1)
-      })
+      }),
+      failAction: async (request, h, error) => {
+        console.log(`approve-application-claim: Error when validating payload: ${error.message}`)
+        const errors = [
+          {
+            text: 'You must select both checkboxes',
+            href: '#authorise-payment-panel'
+          }
+        ]
+        return h
+          .redirect(`/view-application/${request.payload.reference}?page=${request?.payload?.page || 1}&errors=${encodeURIComponent(JSON.stringify(errors))}`)
+          .takeover()
+      }
     },
     handler: async (request, h) => {
-      if (request.payload.approveClaim === 'yes') {
+      if (JSON.stringify(request.payload.confirm) === JSON.stringify(['approveClaim', 'sentChecklist'])) {
```

Comment:
Not sure but I think this would work without the duplicate JSON.stringify?

Reply:
<img width="546" alt="image" src="https://github.com/DEFRA/ffc-ahwr-backoffice/assets/118533775/5fc798f3-a974-4df0-8f4c-e8daaffa0513">